daemon/src/auto_rollover.rs

```diff
@@ -66,28 +72,47 @@ where
         }
     }
 
-    async fn handle(&mut self, Rollover(order_id): Rollover) {
-        let disconnected = match self.rollover_actors.get_disconnected(order_id) {
-            Ok(disconnected) => disconnected,
-            Err(_) => {
-                tracing::debug!(%order_id, "Rollover already in progress");
-                return;
+    async fn handle(
+        &mut self,
+        Rollover {
+            order_id,
+            maker_peer_id,
+        }: Rollover,
+    ) {
+        if let Some(maker_peer_id) = maker_peer_id {
+            if let Err(e) = self
+                .libp2p_rollover
+                .send(ProposeRollover {
+                    order_id,
+                    maker_peer_id,
+                })
+                .await
+            {
+                tracing::error!(%order_id, "Failed to dispatch proposal to libp2p rollover actor: {e:#}");
             }
-        };
+        } else {
```

Comment:
This was mostly for the case where we cannot default the maker peer id - but in production / testnet this should never be a problem and locally it actually does not matter, so yeah, we should remove it :)

Reply:
There is a follow-up PR that removes the old protocol! #2077  (marked as a draft, as we haven't yet got data whether libp2p will be the silver bullet for all our troubles ;-). It can go in shortly after we decide whether libp2p is good enough to keep it.